ad-bank.md

```diff
@@ -0,0 +1,39 @@
+* With PostHog’s self-hosted product analytics solution, your data remains on your servers - ensuring total privacy with no data leaks. Try it out today at posthog.com/product
+* PostHog’s self-hosted product analytics features a Postgres or Clickhouse database for your user data. Manage your server or let us manage it for you hassle-free. Learn more at posthog.com/signup 
+* Our self-hosted product analytics solution features unlimited event ingestions, unlimited user tracking, unlimited data retention, and multiple projects across unlimited teams. Sign up free at posthog.com/signup
+* PostHog: powerful product analytics with SSO, API access, user permissions, and ad blocker resistance. See how it works (and deploy it free) at posthog.com/product 
+
+## POSTHOG CLOUD
+ 
+* PostHog Cloud app analytics comes with automatic upgrades, instant setup, and 1 million free events ingested each month. See how it works at posthog.com/product 
+* PostHog Cloud’s analytics platform gives you accurate insights into how your app is being used so you can build better features. See how it works at posthog.com/product
+* Need a turnkey product analytics solution? PostHog Cloud lets you ingest up to 1m events each month for free and hosts your data on our servers, ensuring maximum uptime. Get started today - no credit card required - at posthog.com/signup 
+* Your product analytics tool should be powerful, easy to use, and affordable. We offer up to 1m ingested events per month - hosted by us or on your servers. See more features at posthog.com/product 
+* PostHog Cloud is a fully-featured product analytics suite that lets you scale as needed at a constant price. Get up to 1m free ingestions per month + more at posthog.com/signup.
+* With PostHog Cloud product analytics, you can ingest unlimited events, track unlimited users, and manage multiple projects with data retained for up to 7 years. Learn more at posthog.com/product. 
+* Learn exactly what your users are doing in your app with PostHog Cloud’s Session Recording feature - now live. See how it works at posthog.com/product.
+* PostHog Cloud’s product analytics suite gives you session recordings, feature flags, integration plugins, and more. Get it today at posthog.com/signup.
+* Finally, a managed product analytics solution that lets you add unlimited members with SSO, API access, and guaranteed uptime. Try out PostHog today at posthog.com/signup.
+* PostHog Cloud lets you sync your product analytics data to Slack, Teams, Discord, and Zapier. Sign up free today at posthog.com/signup 
+* Powerful, open-source product analytics backed by a Slack community and email support? Try PostHog Cloud today at posthog.com/signup and get your first 1m events free each month.
+
+## GENERIC COPY
+
+* PostHog is an analytics tool for mobile apps. Check out our fully featured free tier at posthog.com/signup
```

Comment:
Again, I think this focus on apps limits us and isn't the strongest direction. As a general ad line, I'd prefer:

_PostHog is an analytics platform build for developers and product managers. Try the open source version for free at XXX._